.vscode/settings.json

```diff
@@ -10,5 +10,5 @@
     "source.fixAll.eslint": true
   },
   "eslint.validate": ["typescript"],
-  "editor.formatOnSave": true
-}
+  "editor.formatOnSave": true,
```

Comment:
It'd be great if you could remove the trailing comma, it's not wanted in a JSON file ;)

Reply:
Sorry about that, VSCode loves to meddle with it.

Reply:
no worries, thanks for fixing it